contracts/lib/Organization.sol

```diff
@@ -16,22 +16,51 @@ pragma solidity ^0.5.0;
 
 import "./SafeMath.sol";
 import "./OrganizationInterface.sol";
-import "./IsMemberInterface.sol";
 
 
 /**
  * @title Organization contract.
  *
  * @notice The organization represents an entity that manages the
- *         mosaic ecosystem and therefore the Organization.sol contract holds
+ *         mosaic ecosystem and therefore the `Organization.sol` contract holds
  *         all the keys required to administer the mosaic contracts.
+ *         This contract supports the notion of an "admin" that can act on
+ *         behalf of the organization. When seen from the outside by consumers
+ *         of the `OrganizationInterface`, a notion of an admin does not exist.
  */
-contract Organization is OrganizationInterface, IsMemberInterface {
+contract Organization is OrganizationInterface {
+
+
+    /* Using */
+
     using SafeMath for uint256;
 
+
+    /* Events */
+
+    /** Emitted when a current owner initiates a change of ownership. */
+    event OwnershipTransferInitiated(address indexed proposedOwner);
+
+    /** Emitted when a new owner accepts the ownership transfer. */
+    event OwnershipTransferCompleted(address newOwner);
+
+    /** Emitted whenever an owner or admin changes the address of the admin. */
+    event AdminAddressChanged(address indexed newAdmin);
+
+    /** Emitted when a worker address was set. */
+    event WorkerSet(
+        address indexed worker,
+        uint256 expirationHeight,
+        uint256 remainingHeight
+    );
+
+    /** Emitted when a worker address is deleted from the contract. */
+    event WorkerUnset(address worker, bool wasSet);
```

Comment:
Workers can be set with an expirationHeight 0. To differentiate whether "0 expiration height or non existent worker" is being deleted or " valid worker with non 0 expiration height" is being deleted, wasSet flag was added.

Reply:
If caller needs to differentiate those cases (*if*), it can check it before calling removing worker. Doing it for all costs gas and I believe is not needed for most of our cases.